app/views/reports/regions/show.html.erb

```diff
@@ -228,6 +228,62 @@
   <%= render(Dashboard::Hypertension::LostToFollowUpComponent.new(data: @data, region: @region, period: @period)) %>
 </div>
 
+<div class="d-flex jc-space-between">
+  <h4 class="mt-5 mb-32px">Overdue</h4>
+
+  <div class="d-flex" style="margin-top: 43px;">
+    <form method="get" name="overdue_form">
+      <div class="custom-control custom-switch checkbox-outside-card">
+        <input id="overdueFilter" type="checkbox" class="custom-control-input checkbox-outside-card">
+        <%# <input type="checkbox"
+              onChange="this.form.submit()"
+              name="with_ltfu"
+              for="ltfu_form"
+              class="custom-control-input"
+              id="<%= id %>
+        <%# <label class="font-weight-normal custom-control-label c-grey-dark"
+              for=""> %>
+          <label for="overdueFilter" class="font-weight-normal custom-control-label checkbox-outside-card">
+            Include filtered patients
```

Comment:
Instead of `filtered`, we could write something like `Include non-contactable patients`. What do yo think?
Also, all these strings should be localised.